ironfish/src/syncer.ts

```diff
@@ -373,10 +373,17 @@ export class Syncer {
     let blocksPromise = this.getBlocks(peer, head, this.blocksPerMessage + 1)
 
     while (currentHead) {
+      const blocksResult = await blocksPromise
+      if (!blocksResult.ok) {
+        peer.close()
```

Comment:
Do you want to log the error here like we were doing before? It could be generic too

Reply:
Still logging it in the catch statement above 👍🏻